monai/apps/manifest/config_parser.py

```diff
@@ -176,28 +185,35 @@ def _do_parse(self, config, id: str = ""):
         else:
             self.reference_resolver.add_item(ConfigItem(config=item_conf, id=id))
 
-    def parse(self):
+    def parse(self, reset: bool = False):
         """
-        Recursively parse the config source, add every item as `ConfigItem` to the resolver.
+        Recursively parse the config source, add every item as ``ConfigItem`` to the resolver.
+
+        Args:
+            reset: whether to reset the ``reference_resolver`` before parsing. Defaults to False.
 
         """
+        if reset:
+            self.reference_resolver.reset()
         self._do_parse(config=self.config)
 
-    def get_parsed_content(self, id: str):
+    def get_parsed_content(self, id: str = "", **kwargs):
```

Comment:
added support of `instantiate: bool` and `eval_expr: bool`

Reply:
OK,  I see you also added this `kwargs` for `get_item()` in resolver, I didn't see any `instanatiate=False` use case in `get_item()` because no matter what the `kwargs`, `get_item()` always return the resolved `ConfigItem` instead of the instance or expression result in `resolved_content`.
Anyway, I didn't remove it in this PR, maybe you have some use cases I didn't see?

Thanks.